packages/libraries/router/src/registry.rs

```diff
@@ -76,6 +79,12 @@ impl HiveRegistry {
             }
         }
 
+        if config.schema_file_path.is_none() {
+            if let Ok(schema_file_path) = env::var("HIVE_CDN_SCHEMA_FILE_PATH") {
```

Comment:
I think you should update the [docs](https://the-guild.dev/graphql/hive/docs/integrations/apollo-router) too!

- [ ] Document new env var `HIVE_CDN_SCHEMA_FILE_PATH`
- [ ] Document new config option `schema_file_path` 
- [ ] Add `schema_file_path` to the commented `router.yaml` - not sure if this is a docs only thing or actually part of the code bootstrapping the config too?

Reply:
schema_file_path cant be configured in router.yaml, since this is not a router plugin

Reply:
did update the docs regarding the new env var